src/app/pages/DashboardPage/AverageTransactionSize.tsx

```diff
@@ -4,12 +4,19 @@ import Card from '@mui/material/Card'
 import CardHeader from '@mui/material/CardHeader'
 import CardContent from '@mui/material/CardContent'
 import { LineChart } from '../../components/charts/LineChart'
-import { useGetConsensusStatsTxVolume } from '../../../oasis-indexer/api'
+import { Layer, useGetLayerStatsTxVolume } from '../../../oasis-indexer/api'
 
 export const AverageTransactionSize: FC = () => {
   const { t } = useTranslation()
-  // TODO: Replace with real stats when available
-  const dailyVolumeQuery = useGetConsensusStatsTxVolume()
+  const dailyVolumeQuery = useGetLayerStatsTxVolume(
+    Layer.consensus, // TODO: which layer do we want here?
```

Comment:
We want emerald, but it doesn't work, so consensus + // TODO: switch to emerald

Reply:
ok